L1Trigger/TrackFindingTMTT/src/Settings.cc

```diff
@@ -46,6 +46,7 @@ namespace tmtt {
         kalmanMaxSkipLayersHard_(1),  // On "hard" input tracks
         kalmanMaxSkipLayersEasy_(2),  // On "easy" input tracks
         kalmanMaxStubsEasy_(10),      // Max. #stubs an input track can have to be defined "easy"
+        KFUseMaybeLayers_(false),
```

Comment:
The hard-wired value here determines whether Maybe layers are used by default in the Hybrid. It should be true.